packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -190,14 +203,34 @@ export default util.createRule<Options, MessageIds>({
         return;
       }
 
+      if (ctorCheck === 'off' && isParentConstructor(node)) {
```

Comment:
hmmm - I think I messed up here in reviewing issues, and have lead you astray because of it.

reading #617 again, it looks like it's an invalid request.

Why would disabling the check for constructors also disable checking for parameter properties, when we have a separate option for parameter properties?

The constructor option should only disable checking for the constructor itself.

WDYT? Am I understanding correctly?

Reply:
Exactly! I too thought there's something wrong as I mentioned in my above comment, while writing tests. I'll remove this check totally, as `constructor` is already handled.